From a55153d15581578bb393b81586f3cb98943a4e50 Mon Sep 17 00:00:00 2001
From: Daniel Kurtz <djkurtz@chromium.org>
Date: Wed, 25 Mar 2015 11:55:40 -0700
Subject: [PATCH 4/6] tests: add mediatek to modetest, kmstest, vbltest and
 proptest

There is a mediatek drm kms driver.
Add "mediatek" to the static lists of driver names in the the standard set
of tests.

Signed-off-by: Daniel Kurtz <djkurtz@chromium.org>
Signed-off-by: JB Tsai <jb.tsai@mediatek.com>
---
 tests/kmstest/main.c      | 1 +
 tests/modetest/modetest.c | 2 +-
 tests/proptest/proptest.c | 2 +-
 tests/vbltest/vbltest.c   | 2 +-
 4 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/tests/kmstest/main.c b/tests/kmstest/main.c
index 120bc0f..2baea95 100644
--- a/tests/kmstest/main.c
+++ b/tests/kmstest/main.c
@@ -66,6 +66,7 @@ static const char *drivers[] = {
 	"imx-drm",
 	"rockchip",
 	"atmel-hlcdc",
+	"mediatek",
 	NULL
 };
 
diff --git a/tests/modetest/modetest.c b/tests/modetest/modetest.c
index 3b01918..a0b29e4 100644
--- a/tests/modetest/modetest.c
+++ b/tests/modetest/modetest.c
@@ -1554,7 +1554,7 @@ int main(int argc, char **argv)
 	int drop_master = 0;
 	int test_vsync = 0;
 	int test_cursor = 0;
-	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "exynos", "tilcdc", "msm", "sti", "tegra", "imx-drm", "rockchip", "atmel-hlcdc" };
+	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "exynos", "tilcdc", "msm", "sti", "tegra", "imx-drm", "rockchip", "atmel-hlcdc", "mediatek" };
 	char *device = NULL;
 	char *module = NULL;
 	unsigned int i;
diff --git a/tests/proptest/proptest.c b/tests/proptest/proptest.c
index b9296bd..d5bd1de 100644
--- a/tests/proptest/proptest.c
+++ b/tests/proptest/proptest.c
@@ -303,7 +303,7 @@ static void printUsage(void)
 
 int main(int argc, char *argv[])
 {
-	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "msm", "rockchip" };
+	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "msm", "rockchip", "mediatek" };
 	unsigned int i, ret = 0;
 
 	for (i = 0; i < ARRAY_SIZE(modules); i++){
diff --git a/tests/vbltest/vbltest.c b/tests/vbltest/vbltest.c
index e27f45c..cfc0026 100644
--- a/tests/vbltest/vbltest.c
+++ b/tests/vbltest/vbltest.c
@@ -106,7 +106,7 @@ int main(int argc, char **argv)
 {
 	unsigned i;
 	int c, fd, ret;
-	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "exynos", "omapdrm", "tilcdc", "msm", "tegra", "imx-drm" , "rockchip" };
+	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "exynos", "omapdrm", "tilcdc", "msm", "tegra", "imx-drm" , "rockchip", "mediatek" };
 	drmVBlank vbl;
 	drmEventContext evctx;
 	struct vbl_info handler_info;
-- 
2.1.2

